src/images/svg/playstore/stars/index.js

```diff
@@ -0,0 +1,11 @@
+import Star1 from './star-1.svg'
```

Comment:
Hi @prince-deriv , do we still need this, as we're using the widget now?

Reply:
@prince-deriv Are we removing these star svgs and commenting out the logic for now?

Reply:
@sean-binary  this is the small black stars for Play Store review. It's a ready component waiting for dynamic values from API later